ember-welcome-page/package.json

```diff
@@ -39,11 +50,14 @@
   },
   "devDependencies": {
     "@babel/core": "^7.20.5",
-    "@babel/eslint-parser": "^7.19.1",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-decorators": "^7.20.5",
+    "@babel/preset-typescript": "^7.18.6",
     "@embroider/addon-dev": "^3.0.0",
-    "@rollup/plugin-babel": "^6.0.3",
+    "@tsconfig/ember": "^1.1.0",
+    "@types/ember__application": "^4.0.5",
```

Comment:
Rather than listing all `@types/ember__*` packages that `ember-cli-typescript` installs, I listed only the packages that the source code needs.

For example, `@types/ember__application` was added because `welcome-page.ts` imports `getOwner` from `@ember/application`.

It's possible that I didn't add some `@types/ember__*` packages that should always be included. If so, please let me know.